app/views/templates/inputs/CapturePreferenceHelper.scala.html

```diff
@@ -0,0 +1,27 @@
+@*
```

Comment:
I'd move this to the app/views/helpers package as its not an input, and make it start with lowercase i.e. `capturePreferenceHelper`